client/js/components/month/month.js

```diff
@@ -1,13 +1,13 @@
 /** @jsx React.DOM */
 var React = require('react');
 var MonthEventsContainer = require('./month-events-container.js');
-var MonthStore = require('../../stores/month-store.js');
-var MonthHeader = require('./month-header.js');
-var MonthData = require('../../monthData.js');
+var MonthStore = require('../../stores/month-store');
+var MonthHeader = require('./month-header');
+var MonthData = require('../../monthData');
 
 function monthData () {
   return {
-    gigs: MonthStore.getMonthData()
+    events: MonthStore.getMonthData()
```

Comment:
Having this 'events' property does not match up with the structure of the initial month data. Looking at the MonthEventsContainer component, you expect to loop through the monthWeeks, which will work fine with the initial data Month data the month component receives, but not work with the month data retrieved _onChange. 

```javascript
{
  events: {
                week1: ...
                week2: ...
               ....
             }
}
```